src/os/bluestore/BlueStore.cc

```diff
@@ -8144,6 +8181,7 @@ int BlueStore::_do_alloc_write(
         dblob.add_unused(b_end, wi.blob_length - b_end);
       }
     }
+    b->get_ref(coll.get(), wi.b_off0, wi.length0);
```

Comment:
it looks like this pairs with the special case in get_ref()... can we add a comment for both locations to reference the other?  (e.g., // see _wctx_write_finish())